weatherman/utils.py

```diff
@@ -0,0 +1,44 @@
+import re as regex
+from typing import Tuple
+import os
+
+
+def extract_year_month(date: str) -> Tuple[int, int]:
+    year, month, *_ = map(int, date.split('-'))
+
+    return year, month
+
+
+def extract_year(date: str) -> int:
+    year = int(date.split('-')[0])
+
+    return year
+
+
+def extract_month(date: str):
+    month = int(date.split('-')[1])
+
+    return month
+
+
+def remove_whitespaces_from_start_end(text: str) -> str:
+
+    return text.strip()
+
+
+def is_valid_path(directory_path: str) -> bool:
+
+    return os.path.exists(directory_path)
+
+
+def is_valid_year(date_year: str) -> bool:
```

Comment:
rebase with `main` once you resolve the comments on PR https://github.com/bilal02arbisoft/WeatherMan-App-Task/pull/2